data-model/virtualcollections/v.0.1.0/schemas/virtualcollections.json

```diff
@@ -1 +1,110 @@
-
+{
+  "$schema": "http://json-schema.org/draft-2020-12/schema#",
```

Comment:
Needs additional metadata id and a comment, see for example the digital specimen:
 "$id": "https://schemas.dissco.tech/schemas/digitalobjects/0.1.0/digital-specimens/digital-specimen.json",
 "$schema": "https://json-schema.org/draft/2020-12/schema",
 "$comment":"DigitalObject Version 0.1.0",